doc/source/features.rst

```diff
@@ -48,18 +48,25 @@ only to a morphology or a morphology population.
 
 An example for ``neurite``:
 
-.. code-block:: python
+.. testcode::
 
-   from neurom import load_morphology, features
-   from neurom.features.neurite import max_radial_distance
+    from neurom import load_morphology, features
+    from neurom.features.neurite import max_radial_distance
+
+    m = load_morphology(morphology_path)
+
+    # valid input
+    rd = max_radial_distance(m.neurites[0])
+
+    # invalid input
+    try:
+        rd = max_radial_distance(m)
+    except Exception as e:
+        pass
```

Comment:
Ugly, but I couldn't find a nice way to make this work.

Reply:
Why do we need to show an example with invalid input?
If we really need to keep it, we could comment it out with an explanation, or use the `contextlib.suppress` context manager

Reply:
I like your thinking. I will comment it out!
The why I guess is to show what you can and cannot do with the features imported from different levels (neurite, morphology, population).